### ISSUE TITLE
Activation with a different key

### ISSUE THREAD

**Issue by pietrocavallin** [damienleroy/PowerAccent#55]
Hi Damien,

first of all thank you very much for this fantastic tool. It really makes life easier.

However, especially as a gamer, I find myself telling you that the activation of Quick Accent while playing is not rare. The combination A or E (when using WASD) + spacebar or right/left arrow is very common.

How about adding a third button, or giving the user the option to choose it himself?

Thanks for your attention and keep up the good work!

**Comment by fredgreen182** [comment]
I have to second this, it's been wonderful to use PowerAccent as part of PowerToys, but the lack of a quick way to deactivate or another way to activate it so it doesn't interfere with gaming has definitely been an issue.

**Comment by damienleroy** [comment]
Hello, in last version (of PowerAccent) you have an option to disable the tool when a game is running (btw, please tell me if it's working well for you, I didn't had any return for this, so I guess it is a good news, but I would prefer to be sure).

About the request to have an another option to trigger this, indeed, I will think about that.

**Comment by Darthagnon** [comment]
I wish we could use Caps Lock (or a user-specified hotkey) to trigger the menu. I keep finding myself holding a key for a split-second too long very often while using the computer and it types the accent even before the menu appears. For the moment, I'm going back to [SymWin](https://github.com/Darthagnon/SymWinBinaries/releases), which uses CapsLock to trigger the accent menu more reliably.

**Comment by fredgreen182** [comment]
@damienleroy great to see this added, do you have any control on when these updates get added to the PowerToys version? If so, do you have a rough date on when that might happen?

**Comment by patricklingen** [comment]
Hey, I was just wondering if we can't get the same activation as is handled in OSX, i.e. you just press the character that makes the table appear directly over the cursor - using the arrow and space bar is not a natural workflow.

I currently use Holdkey - https://www.holdkey.eu/

**Comment by damienleroy** [comment]
I agree to add more option to trigger PowerAccent. But options are quite limited, and about Capslock I am not sure. It is use also to create characters with CAPITAL cases. But I need to find the time to test SymWin and know how the behavior is.

**Comment by Darthagnon** [comment]
Related #8 ?

**Comment by jgod** [comment]
I would love to be able to activate it by Right Alt.

**Comment by mabalmer** [comment]
> I would love to be able to activate it by Right Alt.

THIS, please. Having the activation keys limited to space or the left/right arrow keys is _incredibly_ limiting, especially when games are involved. Additionally, the check of having Game Mode being on isn't really much help because so few games actually enable it in the first place (or we've just completely disabled XBox Live integration entirely in Windows because it's a huge pain in the rear) that it may as well not really be a check.
